doc/man1/openssl-verify.pod.in

```diff
@@ -336,45 +337,45 @@ determined.
 
 The verify operation consists of a number of separate steps.
 
-Firstly a certificate chain is built up starting from the supplied certificate
-and ending in the root CA.
-It is an error if the whole chain cannot be built up.
-The chain is built up by looking up the issuers certificate of the current
-certificate.
-If a certificate is found which is its own issuer it is assumed to be the root
-CA.
-
-The process of 'looking up the issuers certificate' itself involves a number of
-steps.
-After all certificates whose subject name matches the issuer name of the current
-certificate are subject to further tests.
+Firstly a certificate chain is built up starting from the target certificate
+and typically ending in a self-signed "root" CA certificate. It is an error if
```

Comment:
Interesting reference. I was not aware of this convention,
which does make sense to me also because it typically gives more room to make small changes within a sentence without affecting the (lines of the) following sentences.

I've re-added the line break here (and in a couple of further places).